task5/src/containers/header/header.js

```diff
@@ -0,0 +1,26 @@
+import React, {useContext} from 'react';
+import './header.scss';
+import LogoHeader from '../../components/logoHeader/logoHeader';
+import Search from '../../components/search/search';
+import { MovieContext } from '../../utils/contexts/movieContext';
+import MovieDetail from '../../components/movieDetail/movieDetail';
+
+function Header() {
+  const {movieId} = useContext(MovieContext);
+
+  return (
+    <div className="header">
+      {
+        movieId ? (<>
+          <MovieDetail/>
```

Comment:
In this case you do not need to use Fragments, we use it when we have more than one element, but in this case it is only one. You can just return `<MovieDetail/>`